src/plots/plots.js

```diff
@@ -576,6 +577,21 @@ plots._hasPlotType = function(category) {
     return false;
 };
 
+// check whether trace has a category
+plots._hasCategory = function(category) {
```

Comment:
I think we should merge this with `hasPlotType`. Base plot module names and categories will never overlap.

Reply:
Done. Although there are possibly some side-effects, needs checking.